config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
+  resources :students
```

Comment:
If you create the controller and the views and you go to the route in the browser you will get an error, so I recommend you add the resources routes in the same commit to avoid those errors.

Also remember to add specs to all your changes 👍